docs/source/conf.py

```diff
@@ -123,7 +124,7 @@
 html_static_path = ["_static", "_templates/_static"]
 
 html_context = {
-    "css_files": [
+    "extra_css_files": [
```

Comment:
Why is this change needed?

Reply:
Because `css_files` is a builtin context variable used by `app.add_css_file` in Sphinx' plugins setup. The copy button provides a css file. If we still want to overwrite `css_files` then we have to duplicate the copy button styles in `ignite_theme.css`.